docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -63,15 +63,10 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
       <td>Read & Write</td>
       <td>Codacy sets the status of commits according to the result of code analysis.</td>
     </tr>
-    <tr>
-      <td>Administration</td>
```

Comment:
@andrzej-janczak Using installation access tokens on GitHub, I believe we don't need the `Administration` permission anymore. Can you please confirm? 🙏

Reply:
Yes, this is true, for sure we drop `write` access 
read access is to be checked in task: https://codacy.atlassian.net/browse/PLUTO-765 
but for 99% will be dropped also, so  I for now I would assume we just drop `Administration`

Reply:
Ok. Let me know when we finish that task. If we keep `Administration` Read permission, I will need to explain how we use that permission.

Reply:
@andrzej-janczak I kept the Administration permission for now describing the fallback mechanism. Let me know if it's correct.

Reply:
LGTM